docs/components/COMPONENT_REPLACEMENT.md

```diff
@@ -132,7 +132,7 @@ This configuration key's value is used at runtime to lookup in the component rep
 
```

Comment:
I cannot put it above, but at the begining, it is a bit difficult to visualize the issue with Angular's replacement component. A comparison of the two implementations would be nice to prove our point

Reply:
In the overview section I think we should start by explaining in one sentence the goal of the feature : make a component replacable at runtime. 
Few other things:
* " the right component at runtime" => "the component at runtime" (right sounds weird in english here IMO
* "Now let's see how to use that mechanism in practice." => Now let's see how to set this mechanism up to create your replacable components.